certified-connectors/Azure Communication Services SMS Events/script.csx

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
```

Comment:
@lei-msft please remove all libraries and files from the script. 
Remove all the "using" definitions. you can start with the class
```
public class Script : ScriptBase
{
    public class StringContainsAdvancedFilter
    {
....
```